easybuild/easyblocks/n/namd.py

```diff
@@ -45,6 +45,7 @@ def extra_options():
             'charm_arch': [None, "Charm++ target architecture", MANDATORY],
             'charm_extra_cxxflags': ['', "Extra C++ compiler options to use for building Charm++", CUSTOM],
             'charm_opts': ['--with-production', "Charm++ build options", CUSTOM],
+            'cuda': [True, "Enable CUDA build if CUDA is among the dependencies", CUSTOM],
```

Comment:
Maybe `None` (unspecified) is a better default?

Then you could:
* if `None`: auto-enable CUDA support if CUDA is listed as a dep
* if `False`: don't enable CUDA, don't even check for CUDA dep
* if `True`: enable CUDA if CUDA is a dep, hard error if CUDA dep is missing

Reply:
Sounds good to me